style.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:wght@300&family=Outfit:wght@300&family=Roboto:ital,wght@0,100;0,300;1,300&family=Water+Brush&display=swap');
+
+@media all and (min-width: 350px) {
+  * {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+    box-sizing: border-box;
+  }
```

Comment:
- [ ] Kindly make sure you are using the correct `font-family` as seen in your chosen design 🙏